### DISCUSSION TITLE
Pants support docs?

### DISCUSSION THREAD

**Discussion by malsharman** [scalacenter/bloop#1273] [General]
Is there any documents/code snippets about how to test out the pants support? Happy to wade through code and help document if I can get some pointers. 
Thanks

**Comment by olafurpg** [comment]
Thank you for reporting! The Pants integration with Bloop currently lives in the Metals repository https://github.com/scalameta/metals and it's undocumented at the moment since it's still evolving and it's unclear what minimum Pants version it will eventually require.
